Vagrantfile

```diff
@@ -41,7 +41,7 @@ source /etc/profile.d/gopath.sh
 SCRIPT
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "chef/ubuntu-12.04"
+  config.vm.box = "ubuntu/trusty64"
```

Comment:
After starting Ubuntu 12.04 there is a note that I should uprade to 14.04. This is annoying. Also I thought it makes sense to use the official Vagrant boxes provided by the Ubuntu project. I was not aware that they do not provide VMware boxes. What about using [chef/ubuntu-14.04](https://atlas.hashicorp.com/chef/boxes/ubuntu-14.04)?